src/main/java/edu/ie3/datamodel/io/factory/typeinput/SystemParticipantTypeInputFactory.java

```diff
@@ -204,7 +210,8 @@ private SystemParticipantTypeInput buildStorageTypeInput(
       double cosPhi) {
     Quantity<Energy> eStorage = data.getQuantity(E_STORAGE, StandardUnits.ENERGY_IN);
     Quantity<Power> pMax = data.getQuantity(P_MAX, StandardUnits.ACTIVE_POWER_IN);
-    Quantity<DimensionlessRate> cprate = data.getQuantity(CP_RATE, StandardUnits.CP_RATE);
+    Quantity<DimensionlessRate> cprate =
```

Comment:
If we keep this I would suggest (needs to be adapted in the constructor below (@ line 221) as well 

```suggestion
    Quantity<DimensionlessRate> activePowerGradient =
```